src/os_posix/os_union_fs.c

```diff
@@ -751,28 +726,29 @@ __union_fs_file_read(
     WT_DECL_RET;
     WT_SESSION_IMPL *session;
     WT_UNION_FILE_HANDLE *union_fh;
-    RW_SERVICE_LEVEL sl;
     char *read_data;
+    bool can_service_read;
```

Comment:
Drop the bool and move the call into the if statement.

Reply:
[f9d7548](https://github.com/wiredtiger/wiredtiger/pull/11268/commits/f9d75486586dd057bad37f040d1d0b58e43dc982)